handbook/README.md

```diff
@@ -50,6 +50,25 @@ For more developed thoughts about __spending guidelines and limits__, please rea
 * It's okay to spend the first minute or two of a meeting to be present and make small talk, if you want.  Being all-remote, it's easy to miss out on hallway chatter and human connections that happen in [meatspace](https://www.dictionary.com/browse/meatspace).  Why not use this time together during the first minute to say "hi".  Then you can jump right in to the topics being discussed?
 * Turning on your camera allows for more complete and intuitive verbal and non-verbal communication.  When joining meetings with new participants who you might not be familiar with yet, feel free to leave your camera on or to turn it off.  When you lead or cohost a meeting, turn your camera on.
 
+### Fleet EE
+
+##### Communicating design changes to Engineering
+For something NEW that has been added to [Figma Fleet EE (current, dev-ready)](https://www.figma.com/file/qpdty1e2n22uZntKUZKEJl/?node-id=0%3A1):
+1. Create a new [GitHub issue](https://github.com/fleetdm/fleet/issues/new)
+2. Detail the required changes (including page links to the relevant layouts), then assign it to the __“Initiatives”__ project.
+
+<img src="https://user-images.githubusercontent.com/78363703/129840932-67d55b5b-8e0e-4fb9-9300-5d458e1b91e4.png" alt="Assign to Initiatives project" width="300"/>
+
+___NOTE:__ Artwork and layouts in Figma Fleet EE (current, dev-ready) are final assets, ready for implementation. Therefore, it’s important NOT to use the “idea” label, as designs in this document are more than ideas - they are something that WILL be implemented._
+
+3. Navigate to the [Initiatives project](https://github.com/orgs/fleetdm/projects/8), and hit “+ Add cards”, pick the new issue, and drag it into the “🤩Inspire me” column. 
+
+<img src="https://user-images.githubusercontent.com/78363703/129840496-54ea4301-be20-46c2-9138-b70bff7198d0.png" alt="Add cards" width="600"/>
+
+<img src="https://user-images.githubusercontent.com/78363703/129840735-3b270429-a92a-476d-87b4-86b93057b2dd.png" alt="Inspire me" width="300"/>
+
+For issues related to something that was ALREADY in Figma Fleet EE (current, dev-ready), and was just __implemented wrong__, e.g, padding/spacing inconsistency etc. Create a [bug issue](https://github.com/fleetdm/fleet/issues/new?assignees=&labels=bug%2C%3Areproduce&template=bug-report.md&title=) and detail the required changes.
```

Comment:
I think "implemented wrong" could use a [Mr. Rogers pass](https://www.theatlantic.com/family/archive/2018/06/mr-rogers-neighborhood-talking-to-kids/562352/)

Reply:
I wonder if folks might overlook this last paragraph about bugs.  Do you think adding a heading like `Unplanned design changes` or `Bugs`, or listing this point about bugs first, could help make it more visible?

Reply:
Yeah, good. I wondered the same thing.